docs/faq.md

```diff
@@ -0,0 +1,33 @@
+# General FAQ
+
+## AMD GPUs crashing
+
+Changing resolution, using DRI_PRIME and doing various other things can cause crashes, but running the command `echo high| sudo tee /sys/bus/pci/drivers/amdgpu/0000:??:??.?/power_dpm_force_performance_level` or adding the `amdgpu.dpm=0` to the kernel commandline stops these crashes.
+
+## The MacBook's screen freezes, fans run up, and then it turns off
+
+This is a T2 panic; try a patched kernel (if you're not using one already), or boot macOS and check the crashlog. You may need a custom ISO. You can also try adding the ``efi=noruntime`` option to GRUB's command line paramaters.
```

Comment:
The broken runtime call crashed linux not the t2, i'm not sure if there are other efi problems that do crash the t2.

If the t2 is crashing, I'd also suggest updating the kernel and apple-bce.